CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# v1.34.0 (2025-03-03)
+
+## OS Changes
+
+* Add support for NVIDIA Multi-Instance GPU (MIG) ([#4418])
+
+## Build Changes
+* Update bottlerocket-core-kit to 6.0.2 ([#4416])
```

Comment:
I'd like to have links to the changelogs to help one confirm what the actual delta is here. For example, I have to check the previous entry to see we went from core kit 6.0.1 -> 6.0.2 and it would be great if there was a link on this line giving me the diff for that repo.